app/services/state_file/ty23_archiver_service.rb

```diff
@@ -1,49 +1,69 @@
 module StateFile
   class Ty23ArchiverService
     INTAKE_MAP = {
-      'az' => StateFileAzIntake,
-      'ny' => StateFileNyIntake,
+      :az => StateFileAzIntake,
+      :ny => StateFileNyIntake,
     }.freeze
 
     attr_reader :state_code, :batch_size, :data_source, :tax_year, :current_batch, :cutoff
 
-    def initialize(state_code:, batch_size: 100, cutoff: '2024-06-01')
+    def initialize(state_code:, batch_size: 100, cutoff: '2025-01-01')
       @state_code = state_code
       @batch_size = batch_size
-      @data_source = INTAKE_MAP[state_code]
+      @data_source = INTAKE_MAP[state_code.to_sym]
       @tax_year = 2023
       @cutoff = cutoff
       @current_batch = nil
-      raise ArgumentError, "#{state_code} isn't an archivable state. Expected one of #{INTAKE_MAP.keys.join(', ')}" unless data_source
+      raise ArgumentError, "#{state_code} isn't an archiveable state. Expected one of #{INTAKE_MAP.keys.join(', ')}" unless data_source
     end
 
     def find_archiveables
-      @current_batch = ActiveRecord::Base.connection.exec_query(query_archiveable)
-      Rails.logger.info("Found #{current_batch.count} #{data_source} intakes to archive.")
+      query_result = ActiveRecord::Base.connection.exec_query(query_archiveable_intakes)
+      @current_batch = query_result.as_json
+      Rails.logger.info("Found #{current_batch.count} #{data_source.name.pluralize} to archive.")
     end
 
     def archive_batch
       archived_ids = []
-      current_batch.each do |record|
-        intake = data_source.find(record['source_intake_id'])
-        archive = StateFileArchivedIntake.new(record.without('source_intake_id'))
-        # TODO: pull mailing address destails off the intake; populate relevant fields on the archived intake record
-        archive.submission_pdf.attach(intake.blob)
-        archive.save!
-        archived_ids << record['source_intake_id']
+      @current_batch&.each do |batch_data|
+        archived_intake = StateFileArchivedIntake.new(hashed_ssn: batch_data['hashed_ssn'])
+        archived_intake.state_code = @state_code
+        archived_intake.tax_year = @tax_year
```

Comment:
prefer you use the accessors for reads; generally good practice to use defined getters/setters over accessing properties directly.